src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -48,7 +48,7 @@ export class GithubAppsLogsComponent implements OnInit {
   isExpansionDetailRow = (i: number, row: Object) => row.hasOwnProperty('detailRow');
 
   constructor(
-    @Inject(MAT_DIALOG_DATA) public matDialogData: string,
+    @Inject(MAT_DIALOG_DATA) public matDialogData: any,
```

Comment:
Should probably replace the `any` with a type, which could be anonymous, maybe something like:
```
@Inject(MAT_DIALOG_DATA) public matDialogData: { value: string | User, getUserEvents: boolean }
```
https://www.typescriptlang.org/docs/handbook/2/objects.html
I could imagine a different structure, where the injected value has two properties `organization` and `user`, and `ngOnInit` takes action based on which is defined.  Might be easier to understand, might not, your call.